submissions/okkkko/js_dom/script.js

```diff
@@ -0,0 +1,80 @@
+    year: " 2007",
+    genres: " Comedy | Drama | Adventure",
+    img: "./img/limited.jpg",
+  },
+  {
+    name: "The Royal Tenenbaums",
+    storyline:
+      "<p>Three grown prodigies, all with a unique genius of some kind, and their mother are staying at the family household. Their father, Royal had left them long ago, and comes back to make things right with his family.</p>",
+    year: " 2001",
+    genres: " Comedy | Drama",
+    img: "./img/the-royal.jpg",
+  },
+  {
+    name: "Fantastic Mr. Fox",
+    storyline:
+      '<p>This is the story of Mr. Fox and his wild ways of hen heckling, turkey taking, and cider sipping, nocturnal, instinctive adventures. He has to put his wild days behind him and do what fathers do best: be responsible. He is too rebellious. He is too wild. He is going to try "just one more raid" on the three nastiest, meanest farmers that are Walter Boggis, Nathan Bunce, and Franklin Bean.</p><p> It is a tale of crossing the line of family responsibilities and midnight adventure and the friendships and awakenings of this country life that is inhabited by Fantastic Mr. Fox and his friends.</p>',
+    year: " 2009",
+    genres: " Comedy | Adventure | Animation",
+    img: "./img/fox.jpg",
+  },
+  {
+    name: "Hotel Chevalier",
+    storyline:
+      "<p>Grief? Depression? Ambiguity in a Paris hotel room. Jack Whitman lies on a bed, ordering a grilled cheese sandwich from room service. His phone rings; it's a woman on her way to see him, a surprise.</p><p> He readies the room, moving without affect, drawing a bath, changing his clothes. She arrives, as does the food, and the complications of their relationship emerge in bits and pieces. He invites her out on the balcony to see his view. Will they make love? Is the relationship over?</p>",
+    year: " 2007",
+    genres: " Short | Drama | Romance",
+    img: "./img/hotel.jpg",
+  },
+];
+const sideMenu = document.querySelector(".menu");
+const iconMenu = document.querySelector(".menu_icon")
+const name = document.querySelector(".name");
+const genres = document.querySelector(".genres");
+const year = document.querySelector(".year");
+const pic = document.querySelector(".pic");
+const text = document.querySelector(".text");
+const iconClick = () => {
+    iconMenu.classList.toggle('open');
+    if (iconMenu.classList.contains('open')) {
+        sideMenu.classList.add('open');
+    }
+    else {
+        sideMenu.classList.remove('open');
+    }
+};
+const btnClick = (event) => {
+  movies.forEach(function (obj) {
```

Comment:
I'd recommend using `Array.prototype.find` here.
This array method can stop the loop when the match is found.